lib/workers/branch/index.ts

```diff
@@ -320,6 +325,74 @@ export async function processBranch(
     } else {
       logger.debug('No updated lock files in branch');
     }
+
+    if (global.trustLevel === 'high') {
+      logger.debug('Checking for post-upgrade hooks');
+      if (
+        is.nonEmptyArray(config.postUpgradeTasks) &&
+        is.nonEmptyArray(global.allowedPostUpgradeTasks)
+      ) {
+        for (const cmd of config.postUpgradeTasks) {
+          if (
+            !_.some(global.allowedPostUpgradeTasks, (pattern: string) =>
+              cmd.match(pattern)
+            )
+          ) {
+            logger.debug(
+              { cmd, allowedPostUpgradeTasks: global.allowedPostUpgradeTasks },
+              'Post-upgrade task did not match any on allowed list'
+            );
+          } else {
+            logger.debug({ cmd }, 'Executing post-change hook');
+
+            const execResult = await exec(cmd, {
+              cwd: config.localDir,
+            });
+
+            logger.debug(execResult, 'Executed post-change hook');
+          }
+        }
+
+        if (is.nonEmptyArray(config.postUpgradeFiles)) {
+          const status = await platform.getRepoStatus();
+
+          for (const filePath of status.modified.concat(status.not_added)) {
+            const relativePath = filePath.substring(config.localDir.length + 1);
+
+            for (const pattern of config.postUpgradeFiles) {
+              if (micromatch.isMatch(relativePath, pattern)) {
```

Comment:
@rarkins I think inverting condition and using `continue` would look better?

Reply:
That would be my preferred approach as well, but the linter complains about this. Since it's only one condition I've left it as it is but if you still prefer that then I'm happy to change it